src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GeometryEngine/SpatialRelationships/SpatialRelationships.xaml.cs

```diff
@@ -0,0 +1,176 @@
+
+            // Create the graphic and add it to the overlay
+            _pointGraphic = new Graphic(point, locationMarker);
+            _graphicsOverlay.Graphics.Add(_pointGraphic);
+
+            // Listen for taps; the spatial relationships will be updated in the handler
+            MyMapView.GeoViewTapped += MapViewTapped;
+
+            // Set the viewpoint to center on the point
+            MyMapView.SetViewpointCenterAsync(point, 200000000);
+        }
+
+        private async void MapViewTapped(object sender, GeoViewInputEventArgs geoViewInputEventArgs)
+        {
+            // Identify the tapped graphics
+            IdentifyGraphicsOverlayResult result = await MyMapView.IdentifyGraphicsOverlayAsync(_graphicsOverlay, geoViewInputEventArgs.Position, 1, false);
+
+            // Return if there are no results
+            if (result.Graphics.Count < 1)
+            {
+                return;
+            }
+
+            // Get the first identified graphic
+            Graphic identifiedGraphic = result.Graphics.First();
+
+            // Clear any existing selection, then select the tapped graphic
+            _graphicsOverlay.ClearSelection();
+            identifiedGraphic.IsSelected = true;
+
+            // Get the selected graphic's geometry
+            Geometry selectedGeometry = identifiedGraphic.Geometry;
+
+            // Update the spatial relationship display
+            switch (selectedGeometry.GeometryType)
+            {
+                case GeometryType.Point:
+                    PointTreeEntry.ItemsSource = null;
+                    PolygonTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _polygonGraphic.Geometry);
+                    PolylineTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _polylineGraphic.Geometry);
+                    break;
+
+                case GeometryType.Polygon:
+                    PolygonTreeEntry.ItemsSource = null;
+                    PointTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _pointGraphic.Geometry);
+                    PolylineTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _polylineGraphic.Geometry);
+                    break;
+
+                case GeometryType.Polyline:
+                    PolylineTreeEntry.ItemsSource = null;
+                    PointTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _pointGraphic.Geometry);
+                    PolygonTreeEntry.ItemsSource = GetSpatialRelationships(selectedGeometry, _polygonGraphic.Geometry);
+                    break;
+            }
+
+            // Expand the tree view
+            PolylineTreeEntry.IsExpanded = true;
+            PolygonTreeEntry.IsExpanded = true;
+            PointTreeEntry.IsExpanded = true;
+        }
+
+        /// <summary>
+        /// Returns a list of spatial relationships between two points
```

Comment:
Clarify some code comments (done for WPF - applies for the other SDK's as well):
```
/// Returns a list of spatial relationships between two geometries
```